docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -57,79 +39,67 @@ namespace Prime.Services
         } 
     }
 }
-
 ```
 
 ### Creating the test project
 
-Next, cd back into the 'unit-testing-using-dotnet-test' directory, and create the `PrimeServices.Tests` directory.
-CD into the `PrimeService.Tests` directory and create a new project using
-`dotnet new xunit`. `dotnet new xunit` creates a test project
-that uses xUnit as the test library. 
+Change the directory back to the *unit-testing-using-dotnet-test* directory and create the *PrimeServices.Tests* directory. The directory structure is shown below:
 
-The generated template configured the test runner
-in the PrimeServiceTests.csproj:
+```
```

Comment:
should we show this after you do the dotnet new xunit so you have the full project structure somewhere?